packages/preset-polaris/src/index.js

```diff
@@ -0,0 +1,43 @@
+import base from '@theme-ui/preset-base'
+
+const text = '"San Francisco", Roboto, "Segoe UI", "Helvetica Neue", sans-serif'
+export const polaris = {
+  ...base,
+  colors: {
+    text: '#454f5b',
+    background: '#fff',
+    primary: '#5C6AC4',
```

Comment:
These hex codes are capitalized inconsistently :) other themes seem to be using all-lowercase